src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -168,16 +177,17 @@ private Cluster findBestProbe(List<Cluster> clusters)
     {
         int highestProbeBitrateBps = 0;
         Cluster bestIt = new Cluster();
-        for (Cluster cluster : clusters) {
+        for (Cluster cluster : clusters)
+        {
             if (cluster.sendMeanMs == 0 || cluster.recvMeanMs == 0)
                 continue;
             if (cluster.numAboveMinDelta > cluster.count / 2 &&
-                    (cluster.recvMeanMs - cluster.sendMeanMs <= 2.0f &&
-                            cluster.sendMeanMs - cluster.recvMeanMs <= 5.0f))
+                (cluster.recvMeanMs - cluster.sendMeanMs <= 2.0f &&
```

Comment:
I would find this easier to read as:

```
                (-2.0f <= cluster.sendMeanMs - cluster.recvMeanMs &&
                          cluster.sendMeanMs - cluster.recvMeanMs <= 5.0f))
```

Because of the  `cluster.` and `MeanMs` prefix and suffix, it is really hard to see that the first difference is the negation of the second.  By multiplying both sides by -1 (and hence inverting the sense of the comparison), it a) becomes much easier to see the symmetry, b) becomes my preferred "in range check".